lib/transform/transformers.js

```diff
@@ -30,7 +30,10 @@ export function assets (asset) {
   transformedAsset.fields.file = reduce(
     asset.fields.file,
      (newFile, file, locale) => {
-       newFile[locale] = omit(file, 'url', 'details')
+       if (newFile[locale].uploadFrom) {
+         delete newFile[locale].uploadFrom
+       }
```

Comment:
hm, but on this case the `newFile` would be left without `uploadFrom` and `upload === undefined` (no .upload and no .url). Shouldn't we keep `uploadFrom` in the `newFile`?